ckanext/ytp/comments/templates/snippets/count_badge.html

```diff
@@ -0,0 +1 @@
+<span class="badge">{{ count }}</span>
```

Comment:
should we have a new line at the end of the file?

Reply:
@duttonw Added new line to end of the file